happypanda/app_constants.py

```diff
@@ -15,38 +15,38 @@
 import os, sys, enum
 import qtawesome as qta
 
-try:
-    import settings
-    from database import db_constants
-except ImportError:
-    from . import settings
-    from .database import db_constants
+from happypanda import settings
+from happypanda.database.arguments import args
+from happypanda.database import db_constants
 
 # Version number
 vs  = '1.0'
 DEBUG = False
 
-OS_NAME = ''
 if sys.platform.startswith('darwin'):
-	OS_NAME = "darwin"
+    OS_NAME = "darwin"
 elif os.name == 'nt':
-	OS_NAME = "windows"
+    OS_NAME = "windows"
+    import ctypes
```

Comment:
If I were to do that I might as well do it for all the other code as well, Pewpews has a few imports in functions and such throughout the codebase.

I basically didn't want it in the namespace if its not being used (this is only for windows)